frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -397,6 +399,57 @@ export default function ClubEditCard({
     {
       name: 'General',
       type: 'group',
+      description: (
+        <div className="mb-4">
+          <a onClick={() => setShowRankModal(true)} className="is-italic">
+            How can filling out this information help your club's ranking?
+          </a>
+          <Modal
+            show={showRankModal}
+            closeModal={() => setShowRankModal(false)}
+            marginBottom={false}
+            width="80%"
+          >
+            <ModalContent className="content mb-4">
+              <h2>Rank Algorithm</h2>
```

Comment:
Nit: can we make this title a little better? Maybe something like "How club rankings are calculated" (you get the idea).